config.kz_default.yaml

```diff
@@ -2,7 +2,7 @@
 #
 # SPDX-License-Identifier: AGPL-3.0-or-later
 
-version: 1.0.0
+version: 0.3.0
```

Comment:
I would probably use `version: 0.4.0` which is the config version corresponding to `0.4.0` release. Have there been any specific reasons to keep `0.3.0`?

Reply:
Thanks for noting it. I have copied the default config of `pypsa-earth` just before `v0.4.0` release. Therefore it is left as `v0.3.0`. I have modified it to match the release version.